test/integration/helpers/settlementModels.js

```diff
@@ -0,0 +1,90 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * ModusBox
+ - Georgi Georgiev <georgi.georgiev@modusbox.com>
+ - Valentin Genev <valentin.genev@modusbox.com>
+ - Nikolay Anastasov <nikolay.anastasov@modusbox.com>
+ - Shashikant Hirugade <shashikant.hirugade@modusbox.com>
+ - Rajiv Mothilal <rajiv.mothilal@modusbox.com>
+ --------------
+ ******/
+
+'use strict'
+
+const Model = require('../../../src/domain/settlement')
+const Enums = require('../../../src/lib/enumCached')
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+const Db = require('@mojaloop/database-lib').Db
+const Cache = require('../../../src/lib/cache')
+const ProxyCache = require('../../../src/lib/proxyCache')
+const ParticipantCached = require('../../../src/models/participant/participantCached')
+const ParticipantCurrencyCached = require('../../../src/models/participant/participantCurrencyCached')
+const ParticipantLimitCached = require('../../../src/models/participant/participantLimitCached')
+const SettlementModelCached = require('../../../src/models/settlement/settlementModelCached')
```

Comment:
These imports need to be checked.

Reply:
Just disregard for now. I swore I had cleaned this up. Will be removed. This is just copy/paste from the kafka int test utils from the CL to jumpstart asserting kafka messages.

Reply:
Removed.